.github/actions/parse-version/action.yml

```diff
@@ -0,0 +1,14 @@
+name: "Parse versions"
```

Comment:
This is a [javascript action](https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action). It uses its own version of node, independent of the job it's running in.